src/module/rules/rule-element/adjust-modifier.ts

```diff
@@ -69,11 +69,12 @@ class AdjustModifierRuleElement extends RuleElementPF2e<AdjustModifierSchema> {
     override beforePrepareData(): void {
         if (this.ignored) return;
 
-        const predicate = new Predicate(this.resolveInjectedProperties(fu.deepClone([...this.predicate])));
```

Comment:
It doesn't seem like this is used anywhere else in the function. Is there a reason this is like that? Optimization? If so it could maybe use a comment.